routes/middleware/authorization.js

```diff
@@ -0,0 +1,27 @@
+const jwt = require('jsonwebtoken');
+const TOKEN_SECRET_KEY = process.env.TOKEN_SECRET_KEY;
+const { ERR_MSG } = require('../../constants/errors/errorMessage');
```

Comment:
require(import) 구문을 파일 최상단에 그루핑하고, 변수 선언은 따로 분리해주세요. 그래야 나중에 보기 좋습니다.

```suggestion
const { ERR_MSG } = require('../../constants/errors/errorMessage');

const { TOKEN_SECRET_KEY } = process.env;
```

글고 사소하긴 한데 process.env에서 디스럭쳐링 할 수 있지 않나 싶네요